src/components/PhoneNumberAuth/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from 'react'
+import firebase, { firebaseErrorResponse } from '../../firebase'
+
+
+function PhoneNumberSuccess(props) {
```

Comment:
Might be better to have this in its own file, if it needs to be its own component at all.

Reply:
it's a little silly :D